openml/runs/functions.py

```diff
@@ -112,6 +122,50 @@ def initialize_model_from_run(run_id):
     run = get_run(run_id)
     return initialize_model(run.setup_id)
 
+def initialize_model_from_trace(run_id, repeat, fold, iteration=None):
+    '''
+    Initialized a model based on the parameters that were set
+    by an optimization procedure (i.e., using the exact same
+    parameter settings)
+
+    Parameters
+    ----------
+    run_id : int
+        The Openml run_id. Should contain a trace file
+
+    repeat: int
+        The repeat nr (column in trace file)
+
+    fold: int
+        The fold nr (column in trace file)
+
+    iteration: int
+        The iteration nr (column in trace file). If None, the
+        best (selected) iteration will be searched (slow)
```

Comment:
Currently, multiple models in the trace can be marked as selected. What happens in that case?